Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -76,8 +76,12 @@ public void setWordCount(String url, int wordCount) {
 
   public void insertScriptedData(String url) {
     SQLiteDatabase dataBase = getWritableDatabase();
-    dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
-    dataBase.close();
+    try {
+      dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
+      dataBase.close();
+    } catch (Exception exception) {
+      dataBase.close();
+    }
```

Comment:
I add new method. It is checking url in every url insert.
and set back insertScriptedData method (no try catch)